skill-tree /src/main/java/com/RDS/skilltree/Endorsement/EndorsementModel.java

```diff
@@ -0,0 +1,44 @@
+package com.RDS.skilltree.Endorsement;
+
+import com.RDS.skilltree.Skill.SkillModel;
+import com.RDS.skilltree.User.UserModel;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@Getter
```

Comment:
Explore if data annotation can be used in place of getter and setter.

Reply:
changed